corehq/form_processor/management/commands/fix_edit_forms_attachments.py

```diff
@@ -32,13 +32,15 @@ def get_sql_previous_versions(form_id, form_accessor):
 
 
 class Command(BaseCommand):
+    def __init__(self):
+        self.forms = {}
```

Comment:
this should call `super(Command, self).__init__()`, and probably pass `*args, **kwargs`